EngineLayer/NonSpecificEnzymeSearch/NonSpecificEnzymeSearchEngine.cs

```diff
@@ -145,63 +229,203 @@ protected override MetaMorpheusEngineResults RunSpecific()
             return new MetaMorpheusEngineResults(this);
         }
 
-        private Tuple<int, double> Accepts(double scanPrecursorMass, CompactPeptide peptide, TerminusType terminusType, MassDiffAcceptor searchMode)
+        //public static IEnumerable<Product> Fragment( DissociationType dissociationType, FragmentationTerminus fragmentationTerminus, PeptideWithSetModifications pwsm)
```

Comment:
I think this was here for debugging purposes and should be deleted. Zach?

Reply:
Yes, delete